Ajout CRUD Type de Traitement ET traitement

PatateHunter opened the pull request:
Ajout Verification si user est vétérinaire
Ajout Classe Utilitaire DateHelp

PatateHunter committed:
Ajout CRUD Type de Traitement ET traitement
Ajout Verification si user est vétérinaire
Ajout Classe Utilitaire DateHelp

mlle-alois committed:
Update give-access.ts

mlle-alois committed:
Update treatment-controller.ts

mlle-alois committed:
Update treatment-router.ts

Kaseriu reviewed (commented):
Corriger la doc parlant de types d'espace à la place de types de traitement des fonctions du fichier treatment-type-router.ts

PatateHunter committed:
Ajout CRUD Type de Traitement ET traitement
Ajout Verification si user est vétérinaire
Ajout Classe Utilitaire DateHelp
Ajout méthode static ifXXXExist dans Animal, User et treatment-type.
Clean de commentaire
Ajout de controle lors de la modification d'une présence

PatateHunter committed:
Merge remote-tracking branch 'origin/ZOO-0012' into ZOO-0012

# Conflicts:
#	controllers/treatment-controller.ts

PatateHunter committed:
Modif commentaire

PatateHunter committed:
Modif commentaire 2

mlle-alois committed:
Update treatment-router.ts
